store/build.xml

```diff
@@ -298,6 +298,9 @@
       <param name="jar.build.dir" value="${server.dir}/build/test-classes/"/>
     </antcall>
   </target>
+  <target name="publish-store-test" depends="test-jar">
+      <copy file="${build.dir}/zimbrastore-test.jar" tofile="${user.home}/.zcs-deps/zimbrastore-test-1.0.jar"/>
```

Comment:
build-ivysettings.xml refers to ${dev.home}/.zcs-deps - suggest using that for consistency.